### PR TITLE
update vep config etc

### LINKED ISSUE #32: Add support for iGenomes/local genome store
### Description of feature

So we avoid having to run all the reference input steps.

cfr https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-preprocessing/blob/main/nextflow.config#L28-L34

### PR DESCRIPTION
**Pull request by matthdsm**
- Update VEP settings cfr current bcbio config
- bump modules
- bring workflow naming in line with preprocessing
- Add support for iGenomes/local genomes config, closes #32 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-germline/blob/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/samtools_merge.nf [hunk 1]
```diff
@@ -26,7 +26,7 @@ process SAMTOOLS_MERGE {
     def args2           = task.ext.args2   ?: ''
     def prefix          = task.ext.prefix ?: "${meta.id}"
     def reference       = fasta ? "--reference ${fasta}" : ""
-    def convert_to_cram = always_use_cram ?
```
Comment (matthdsm) [inline modules/local/samtools_merge.nf:29]: @nvnieuwk , what's the point of this? The original module also supports cram.

**Comment by matthdsm** [comment]
@nvnieuwk , I'm going to merge this PR into your v1.1.0dev branch if the local tests pass. Can you give a thumbsup if you're okay with that? I'm going to continue work in your branch while you're away.

**Comment by nvnieuwk** [comment]
I'll have a quick look now, I have a local change that overhauls all the tests, but I am waiting for the new update of `nf-test` to push it (because it won't work right now)

File: modules/local/samtools_merge.nf (hunk 1 above)
Comment (nvnieuwk) [reply modules/local/samtools_merge.nf:29]: That was for when the input file is a BAM file. That argument could be specified to convert it to CRAM

**Comment by matthdsm** [comment]
can always add the tests to your branch from another pr

**Review by nvnieuwk — APPROVED** [review]
LGTM, make sure to always have a functional pipeline in the `1.1.0dev` branch, because that one is currently used to test the pipeline on the HPC (but there probably won't be any runs before I'm back and the end of the maintenance period)

**Comment by matthdsm** [comment]
Local tests pass, so I think it's good to go!

**Pull request merged**